docs/key-issues/multiple_architectures.md

```diff
@@ -117,26 +117,44 @@ and ARM64 slices.
     not provide them). There are some arguments for and against supporting the
     format or even defaulting to it.
 
-    Arguments for (Russell to write):
-
-    - xxx
+    Arguments for:
+
+    - While users with a technical background are usually aware of the CPU
+      architecture of their machine, less technical end users are often
+      unaware of this detail (or of the significance of this detail). The
+      universal2 wheel format allows users to largely ignore this detail, as
+      all CPU architectures for the macOS platform are accomodated in a single
+      binary artefact.
+    - macOS has developed an ecosystem where end users expect that any macOS
+      binary will run on any macOS machine, regardless of CPU architecture.
+      As a result, when building macOS apps (`.dmg` downloadable installers
+      or similar formats, produced by tools such as
+      [py2app](https://py2app.readthedocs.io) or
+      [briefcase](https://beeware.org/project/projects/tools/briefcase/)),
+      the person building the project must be accommodate all possible CPU
+      architectures where the code *could* be executed.
+    - If binary wheels are only available in "thin" format, any issues with
+      merging those wheels into fat equivalents for distribution purposes are
+      deferred to the end user. This can be problematic as it may require
```

Comment:
End user as described in this system is people installing apps, and distinguished from "person building the project".

The issues with merging the wheels into fat equivalents is surely not deferred to the end user? It is deferred to the person building the "macOS apps" project.

Reply:
Agreed the usage of "end user" is ambiguous here. I'll clean that up.